auslib/test/test_db.py

```diff
@@ -2729,6 +2729,89 @@ def testInitAndIncrementValue(self):
         self.assertEqual(entry['watchdog'], 2)
 
 
+class TestShutoffs(unittest.TestCase, MemoryDatabaseMixin):
```

Comment:
These tests will need a few updates after the schema changes. Please make sure you explicitly test cases where channel is "*".